### DISCUSSION TITLE
better mouse controls

### DISCUSSION THREAD

**Discussion by sludgeXP** [Northfear/fallout2-ce-vita#10] [Ideas]
sometimes its difficult to hit buttons using the analog stick because theyre so small, and i have the same issue with the touch screen sometimes, and it also feels a little awkward needing to move my finger all over the screen and then move it back to the controller. the way that they use the mouse in something like the daedalusx64 ui (the touch screen being more like a trackpad)  is a lot more comfortable and feels a lot more natural to use. really great work on the port though, everything runs great and i havent run into anything game breaking so far

https://user-images.githubusercontent.com/104333527/183278585-e7bdcc79-a926-46d7-a2bf-c7f90641152d.mp4

**Comment by Northfear** [comment]
It's already in. Check the "Touch Controls" section of readme file

**Reply by sludgeXP** [reply]
well now i feel silly

**Reply by Northfear** [reply]
No worries. It was added recently and README was kinda vague.

**Reply by sludgeXP** [reply]
sorry if this is a tone deaf question, but where/how do i actually change it? im really new to this sort of thing

**Reply by Northfear** [reply]
Open ```ux0:data/fallout2/f2_res.ini``` file (With VitaShell for example. Or edit it from your PC) and change ```FRONT_TOUCH_MODE=1``` line to ```FRONT_TOUCH_MODE=2``` in it

**Reply by sludgeXP** [reply]
thanks a ton, this is awesome
